weld/llvm.rs

```diff
@@ -949,22 +847,34 @@ impl LlvmGenerator {
             Scalar(F64) => Ok("double"),
 
             Simd(Bool) => {
-                Ok(self.vectorized_names.entry(Bool).or_insert(format!("<{} x i1>", vec_size(&Scalar(Bool))?)))
+                Ok(self.vectorized_names
+                       .entry(Bool)
+                       .or_insert(format!("<{} x i1>", vec_size(&Scalar(Bool))?)))
```

Comment:
These looked better on 1 line, what's wrong? maybe we need to bump it up to 110 or 120